master/custom/factories.py

```diff
@@ -276,14 +281,34 @@ def setup(self, parallel, branch, **kwargs):
         if parallel:
             test_command.append(parallel)
         self.addStep(Compile(command=build_command))
-        self.addStep(
-            ShellCommand(
-                name="pythoninfo",
-                description="pythoninfo",
-                command=self.python_command + ["-m", "test.pythoninfo"],
-                warnOnFailure=True,
+        if self.remoteTest:
+            # deploy
+            self.addStep(
+                ShellCommand(
+                    name="remotedeploy",
+                    description="remotedeploy",
+                    command=self.remote_deploy_command + self.remoteDeployFlags,
+                    warnOnFailure=True,
+                )
+            )
+            # pythonInfo
+            self.addStep(
+                ShellCommand(
+                    name="remotepythoninfo",
+                    description="remotepythoninfo",
+                    command=self.remote_pythonInfo_command + self.remotePythonInfoFlags,
+                    warnOnFailure=True,
+                )
+            )
+        else:
+            self.addStep(
+                ShellCommand(
+                    name="pythoninfo",
+                    description="pythoninfo",
+                    command=self.python_command + ["-m", "test.pythoninfo"],
```

Comment:
This line doesn't call any .bat files.  It ends up executing an arm32 binary on the x86/x64 build host which doesn't work